scripts/benchmark_stats.py

```diff
@@ -89,6 +102,16 @@ def _process_file_data(self):
                         )
                         self._requests[row["Type"]]["total"] += request_count
 
+    @property
+    def pdf_percentile(self) -> Any:
+        if self._pdf_percentile:
+            return int(sum(self._pdf_percentile) / len(self._pdf_percentile))
```

Comment:
`int(x)` only crops decimals so better if we use `round(x)` or if you want to math.floor(x) or math.ceil(x), but round(x) does both depending on the situation.

Reply:
Updated 👍